packages/client/src/app/models/site/port-works.ts

```diff
@@ -3,5 +3,5 @@ import { Work } from '@pulp-fiction/models/works';
 
 export interface PortWorks {
     works: PaginateResult<Work>,
-    userId: string;
+    userWorks: PaginateResult<Work>
```

Comment:
What's the difference between works and userWorks?

Reply:
`works` is for the published, content-filtered stuff that anyone other than the author would see
`userWorks` is for all a user's owned works

they both need to come along so that the page displays properly